cms/tests/test_placeholder_admin.py

```diff
@@ -134,14 +135,42 @@ def test_edit_plugin_endpoint(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(plugin.name, data['name'])
 
+    def test_move_plugin_endpoint(self):
```

Comment:
remove this test

Reply:
Again I had already removed these. Merging the latest 4 to allow me to create a PR must have changed all of this back.